### PR TITLE
Bug push stop working losing connection

### LINKED ISSUE #1518: Alarm stops working after switching from 4G to WiFi
Apparently, the alpha testers push alarm stopped working after moving from 4G network to WiFi network, and he needed to restart the app again to get it working.

### PR DESCRIPTION
**Pull request by manuelplazaspalacio**
Close #1518

### CODE REVIEW

File: app/src/ereferrals/java/org/eyeseetea/malariacare/domain/usecase/LoginUseCase.java [hunk 1]
```diff
@@ -148,7 +146,9 @@ private void pullOrganisationCredentials() {
                     true);
         }
 
-        mCredentialsLocalDataSource.saveOrganisationCredentials(orgUnitCredentials);
+        if (orgUnitCredentials != null) {
```
Comment (xurxodev) [inline app/src/ereferrals/java/org/eyeseetea/malariacare/domain/usecase/LoginUseCase.java:149]: if orgUnitCredentials can be null if network fail to retrieve credentials from the server then in the checkUserCredentialsWithOrgUnit method we should validate null

**Review by xurxodev — CHANGES_REQUESTED** [review]
@manuelplazaspalacio I have requested you some changes, can you review it please?

**Comment by manuelplazaspalacio** [comment]
Changes done @xurxodev

**Review by xurxodev — APPROVED** [review]
thanks @manuelplazaspalacio

**Pull request merged**